python-frontend/src/main/java/org/sonar/python/types/TypeShed.java

```diff
@@ -72,7 +73,7 @@ private TypeShed() {
   }
 
   public static Map<String, Symbol> builtinSymbols() {
-    if (TypeShed.builtins == null) {
+    if (TypeShed.builtins == null && !InferredTypes.isInitialized()) {
```

Comment:
can you add a comment explaining why do we need this `InferredTypes.isInitialized`?